SCClassLibrary/Common/Control/Server.sc

```diff
@@ -331,6 +347,8 @@ Server {
 
 	}
 
+	numUsers { ^options.maxLogins }
```

Comment:
numClients?
I'd generally avoid talking about "Users". We don't know how peeps will use this system and how many "Users" there will be. Client is the technical term, therefore I say stick with it.